web/server/api/endpoints/plan.py

```diff
@@ -33,6 +34,12 @@ async def run_plan(
             origin="API -> plan -> run_plan",
         )
 
+    tracker = models.PlanApplyStageTracker(
+        environment=environment, options={"skip_tests": plan_options.skip_tests}
```

Comment:
Is this right? Should options be plan_options instead? Something like `plan_options=models.PlanOptions(skip_tests=plan_options.skip_tests` or even `plan_options=plan_options`?

Reply:
yeah, i reverted my previous change and it removed `plan_options=plan_options`